src/Microsoft.AspNetCore.OData/ODataValueProviderFactory.cs

```diff
@@ -5,7 +5,10 @@
 using System.Threading.Tasks;
 using Microsoft.AspNet.OData.Common;
 using Microsoft.AspNet.OData.Extensions;
-using Microsoft.AspNetCore.Mvc.Internal;
+#if NETCOREAPP3_0
+#else
```

Comment:
Rather than check for NETCOREAPP3_0 (which would break when targeting 3_1 or 5, would prefer:

#if NETSTANDARD2_0
   using Microsoft.AspNetCore.Mvc.Internal
#endif